zcash_primitives/src/transaction/tests.rs

```diff
@@ -36,6 +39,7 @@ fn tx_read_write() {
     assert_eq!(&data[..], &encoded[..]);
 }
 
+#[cfg(test)]
```

Comment:
It would be nice to export `transaction::tests::data` from lib directly instead of adding everything in `transaction::tests` into compilations with the `test-dependencies` feature enabled and hiding most of it individually.